sycl/source/detail/kernel_compiler/kernel_compiler_sycl.cpp

```diff
@@ -165,61 +162,71 @@ std::string getCompilerName() {
   return Compiler;
 }
 
-void invokeCompiler(const std::filesystem::path &FPath,
-                    const std::filesystem::path &DPath, const std::string &Id,
-                    const std::vector<std::string> &UserArgs,
-                    std::string *LogPtr) {
+int invokeCommand(const std::string &command, std::string &output) {
+#ifdef _WIN32
+  FILE *pipe = _popen(command.c_str(), "r");
+#else
+  FILE *pipe = popen(command.c_str(), "r");
+#endif
+  if (!pipe) {
+    return -1;
+  }
+
+  char buffer[124];
```

Comment:
It's just a nit, but 124 is a weird size, any reason to choose it?

Reply:
Initially I had it at 1024. I must have deleted that 0 at some point. I have more kernel_compiler work upcoming, I'll restore it there.